Sources/ArcGISToolkit/Components/Form/ClearButton.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+/// A circular button with a cross in the center, intended to be used to clear text fields.
+struct ClearButton: View {
+    /// The action to be performed when the button is pressed.
+    let action: () -> Void
+    
+    var body: some View {
+        Button {
+            action()
+        } label: {
+            Image(systemName: "x.circle.fill")
```

Comment:
```suggestion
            Image(systemName: "xmark.circle.fill")
```

`xmark.circle.fill` is the one used by `SearchField` and it appears the one used by other iOS apps. There is a subtle difference...

_x.circle.fill_

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/108b028c-81dc-4de8-b117-aaf1f85f85ea)

_xmark.circle.fill_

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/cd8a067b-2eb8-4903-a784-631bac7464db)